lib/OpenQA/WebAPI/Plugin/Helpers.pm

```diff
@@ -317,6 +317,22 @@ sub register {
             my ($c, $group) = @_;
             return $c->tag('li', $c->link_to($group->name => $c->url_for('group_overview', groupid => $group->id)));
         });
+
+    $app->helper(
+        comment_icon => sub {
+            my ($c, $jobid, $comment_count) = @_;
+            return '' unless $comment_count;
+
+            return $c->link_to(
+                $c->url_for('test', 'testid' => $jobid) . '#comments' => sub {
```

Comment:
I'm aware, but this is already fixed in another PR: https://github.com/os-autoinst/openQA/pull/1388/files#diff-ce6d98ac784b7abc299796cba21eba43R331